app/assets/modules/Console/index.tsx

```diff
@@ -85,17 +87,62 @@ class Console extends React.Component<IProps, IState> {
     if (sentenceList.some(sentence => sentence.match(reg))) {
       return message.error(intl.get('common.disablesUseToSwitchSpace'));
     }
+    const { normalSentenceList, paramsMap } = this.handleParamDSL(sentenceList);
     this.editor.execCommand('goDocEnd');
     const history = this.getLocalStorage();
     history.push(gql);
     localStorage.setItem('history', JSON.stringify(history));
 
-    await this.props.asyncRunGQL(gql);
+    await this.props.asyncRunGQL({
+      gql: normalSentenceList.join(';'),
+      paramsMap,
+    });
     this.setState({
       isUpDown: true,
     });
   };
 
+  // 处理:param 语法
+  handleParamDSL = (sentenceList: string[]) => {
+    const reg = /^\s*:PARAM \S+\s*=>\s*\S;?/gm;
+    const paramList: string[] = [];
+    const normalSentenceList: string[] = [];
+    sentenceList.forEach(sentence => {
+      if (sentence.match(reg)) {
+        paramList.push(sentence);
+      } else {
+        normalSentenceList.push(sentence);
+      }
+    });
+    const sessionStorage = getSessionStorage();
+    const paramsMap = JSON.parse(sessionStorage?.getItem('paramsMap') || '{}');
```

Comment:
in this case, it will throw error
```
sessionStorage.setItem('test', undefined);
/** sessionStorage.getItem('test') --> 'undefined' */
JSON.parse(sessionStorage.getItem('test'));
```
sometimes it's better to use a `safeParse` method in which `JSON.parse` is wrapped by `try {} catch() {}`
```
function safeParse<T extends unknown>(s: string): T | undefined {
  try {
    return JSON.parse(s);
  } catch (error) {
    /** error info */
    console.error(`JSON parse error`, error);
    return undefined;
  }
}

const test = safeParse<string>('{}')
```

Reply:
done